.github/workflows/aws-experiment-pipeline.yml

```diff
@@ -0,0 +1,261 @@
+---
+name: AWS-Experiment-Pipeline
+on:
+  workflow_dispatch:
+    inputs:
+      goExperimentImage:
+        default: "litmuschaos/go-runner:ci"
+      operatorImage:
+        default: "litmuschaos/chaos-operator:ci"
+      runnerImage:
+        default: "litmuschaos/chaos-runner:ci"
+      chaosNamespace:
+        default: "default"
+      experimentImagePullPolicy:
+        default: "Always"
+  
+jobs:
+  AWS_EC2_Terminate_By_ID_And_Tag:
+    runs-on: ubuntu-latest
+    steps:
+    
+      #Install and configure a k3s cluster
+      - name: Installing Prerequisites (K3S Cluster)
+        env: 
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml
+        run: |
+          curl -sfL https://get.k3s.io | sh -s - --docker --write-kubeconfig-mode 664
+          kubectl wait node --all --for condition=ready --timeout=90s
+          mkdir -p $HOME/.kube && cat /etc/rancher/k3s/k3s.yaml > $HOME/.kube/config          
+          kubectl get nodes
+          
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.14'
+
+      - name: Create Kubernetes secret for aws experiment
+        if: always()
+        env:
+          KUBECONFIG: /etc/rancher/k3s/k3s.yaml      
+        run: |
+          cat <<EOF | kubectl apply -f -
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: cloud-secret
+            namespace: default
+          type: Opaque
+          stringData:
+            cloud_config.yml: |-
+              [default]
+              aws_access_key_id = ${{ secrets.AWS_ACCESS_KEY_ID }}
+              aws_secret_access_key = ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          EOF
+          
+      - name: Configure AWS Credentials
+        if: always()
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          # aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }} # if you have/need it
+          aws-region: us-west-1
+          
+      - name: Creating target EC2 instances
+        if: always()
+        run: |
+          aws ec2 run-instances \
+          --image-id ami-0d382e80be7ffdae5 \
+          --count 2 --instance-type t2.small \
+          --key-name udit-litmus-e2e \
+          --security-group-ids sg-06925d6218d07e795 \
+          --subnet-id subnet-9c2ba8fa \
```

Comment:
Are the security group and subnet already created or these are the default ones?

Reply:
These are default ones